pkg/component/nodemanagement/machinecontrollermanager/machine_controller_manager_test.go

```diff
@@ -524,10 +583,19 @@ subjects:
 			serviceAccount.ResourceVersion = "1"
 			Expect(actualServiceAccount).To(Equal(serviceAccount))
 
+			//TODO: @aaronfern Remove this after g/g:v1.119 is released
```

Comment:
```suggestion
			// TODO(aaronfern): Remove this after v1.120 is released.
```

Nit: You don't have such TODO in the `cluster_autoscaler_test.go`.